### ISSUE TITLE
Invalid output at $TERM=xterm-256color

### ISSUE THREAD

**Issue by 0xF6** [cake-build/cake#2205]
#### What You Are Seeing?
Invalid output

#### What is Expected?
Valid output

#### What version of Cake are you using?
0.28.1

#### Are you running on a 32 or 64 bit system?
64x

#### What environment are you running on? Windows? Linux? Mac?
Linux

#### How Did You Get This To Happen? (Steps to Reproduce)

Script:
```CSharp
var target = Argument("target", "Default");

Task("Default")
  .Does(() =>
{
  Information("Hello World!");
});

RunTarget(target);
```

`./build.sh`

Result:

![image](https://user-images.githubusercontent.com/13326808/41950352-6d9966b0-79ce-11e8-8dbf-7c84e1d1eb82.png)

## Software

dotnet 2.1.300

![image](https://user-images.githubusercontent.com/13326808/41950446-cbfa624a-79ce-11e8-8348-5514b12b1f09.png)

**Comment by 0xF6** [comment]
`TERM=xterm-256color`
![image](https://user-images.githubusercontent.com/13326808/42075628-69ea2510-7b7a-11e8-963a-aedc15d6fdd1.png) 
`TERM=xterm`
![image](https://user-images.githubusercontent.com/13326808/42075638-7b3a086c-7b7a-11e8-9102-fc313165798c.png)

**Comment by devlead** [comment]
While I haven't seen this myself, wondering if this might be a Mono issue, coul you try and see if you get the same running under .NET Core? 

Example project you could try;
https://github.com/csharpfritz/CoreWiki/

Also which version of Mono are you using? If it's a Mono issue it might have been resolved in a newer version.

**Comment by 0xF6** [comment]
![image](https://user-images.githubusercontent.com/13326808/42085504-464cdb6a-7b99-11e8-94b4-4a30dccc83ce.png)

**Comment by MiddTech** [comment]
Can confirm this is also happening on manjaro [ .net core 2.1.300 | mono 5.12 ] 

There is an open issue possibly to do with an ncurses update that broke some mono interactions
https://github.com/mono/mono/issues/8340

a possibly related (but closed) .net core issue 
https://github.com/dotnet/corefx/issues/26966

**Comment by patriksvensson** [comment]
@0xF6 Seems to be fixed in https://github.com/mono/mono/pull/9520
Can you see if this solves your issue?

**Comment by 0xF6** [comment]
I'm waiting mono update
![image](https://user-images.githubusercontent.com/13326808/44185787-472ca880-a11e-11e8-997b-f7bc62416dcc.png)

**Comment by 0xF6** [comment]
at TERM=xterm-256color:
![image](https://user-images.githubusercontent.com/13326808/44297543-fe225300-a2db-11e8-8db9-76a344f18b65.png)
Mono: `Mono JIT compiler version 5.14.0`
background color has broken? Or is that how it should be?

**Comment by augustoproiete** [comment]
@0xF6 The background color has been fixed in Cake v1.1.0 via https://github.com/cake-build/cake/pull/3094
